### PR TITLE
Prepare for review on https://goreportcard.com

### PR DESCRIPTION
**Pull request by ivanilves**

### CODE REVIEW

**Comment by vonrabbe** [comment]
What a GIF :clap:

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ go: 1.8.3
 
 env:
   global:
-  - RUN_SLOW_TESTS=true
+  - SKIP_SLOW_TESTS=false
```
Comment (vonrabbe) [inline .travis.yml]: Run all tests! Always! YES!

File: Dockerfile [hunk 2]
```diff
@@ -4,6 +4,7 @@
 FROM golang:1.8 AS builder
 ENV DOCKER_HOST tcp://172.17.0.1:2375
 ENV LOCAL_REGISTRY 172.17.0.1
+ENV SKIP_SLOW_TESTS true
```
Comment (vonrabbe) [inline Dockerfile]: you are pussy @ivanilves !

Comment (ivanilves) [reply Dockerfile]: that's bulling @vonrabbe :trollface: 

I'm so offended!

**Pull request merged**